pom.xml

```diff
@@ -920,7 +920,7 @@
         <spring-web.version>4.3.29.RELEASE</spring-web.version>
         <swagger-jaxrs.version>1.6.2</swagger-jaxrs.version>
         <!-- httpclient dependency Version-->
-        <http.client.version>4.3.6.wso2v1</http.client.version>
+        <http.client.version>4.5.13.wso2v1</http.client.version>
```

Comment:
don't we need to update the http core version?

Reply:
Will update the http core to the latest version

Reply:
is this addressed?

Reply:
yes,  https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1830/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R924